YandexLogin/YandexLogin/YandexPages/MailPage.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+
+namespace YandexPageObject.YandexPages
+{
+    public class MailPage
+    {
+        private WebDriver driver;
+        public MailPage(WebDriver driver) { this.driver = driver;}
+        IWebElement userLabel => driver.FindElement(By.ClassName("desk-notif-card__title"));
```

Comment:
Start your Web Elements from the capital letter

Reply:
Done